app/robots.txt

```diff
@@ -2,4 +2,20 @@
 layout: null
 ---
 User-agent: *
+Disallow: /enterprise/0*
+Disallow: /enterprise/1*
+Disallow: /enterprise/2.1.x/
+Disallow: /enterprise/2.2.x/
+Disallow: /enterprise/2.3.x/
+Disallow: /mesh/1.0*
+Disallow: /mesh/1.1*
+Disallow: /gateway-oss/0*
+Disallow: /gateway-oss/1*
+Disallow: /gateway-oss/2.0*
+Disallow: /gateway-oss/2.1*
+Disallow: /gateway-oss/2.2*
+Disallow: /gateway-oss/2.3*
+Disallow: /kubernetes-ingress-controller/1.0*
+Disallow: /kubernetes-ingress-controller/1.1*
+
```

Comment:
I'd also add the getting started guide here:

```suggestion
Disallow: /getting-started-guide/CE*
Disallow: /getting-started-guide/2.0*
Disallow: /getting-started-guide/2.1*
Disallow: /getting-started-guide/2.2*
Disallow: /getting-started-guide/2.3*
```

Reply:
if we're talking about Google specifically, someone can control the crawl using Google's tools.

https://support.google.com/webmasters/answer/48620?hl=en

I believe Google crawls every few days. So there potentially could be an overlap on new versions but it wouldn't last long.